packages/react-player/src/react-player.tsx

```diff
@@ -163,6 +167,12 @@ export const DotLottiePlayer: React.FC<DotLottiePlayerProps> = ({
     }
   }, [activeAnimationId]);
 
+  useUpdateEffect(() => {
+    if (typeof activeStateId !== 'undefined') {
+      dotLottiePlayer.enterInteractiveMode(activeStateId);
+    }
+  }, [activeStateId]);
```

Comment:
don't we need to exit interactivity mode on cleanup ? 

```js
useUpdateEffect(() => {
    if (typeof activeStateId !== 'undefined') {
      dotLottiePlayer.enterInteractiveMode(activeStateId);
    }
    
    return () => { 
      // do something on mount -> exit Interactivity Mode ?? 
    }
  })
  ```

Reply:
@afsalz thoughts?

Reply:
@samuelOsborne @theashraf Yeah I think we need to add this. ill add that in.

Reply:
https://github.com/dotlottie/player-component/pull/198